airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -142,8 +155,15 @@ def _fetch_dag_runs(
     )
     # Count items
     total_entries = query.count()
+    # sort
+    if sort == 'asc':
+        query = query.order_by(asc(order_by))
+    elif sort == 'desc':
+        query = query.order_by(desc(order_by))
+    else:
+        query = query.order_by(DagRun.id)
```

Comment:
I’d probably also handle `sort="desc", order_by=None` here by ordering desc by ID; same aplies for other changes. (Probably a good idea to somehow extract the logic to a function?)

Reply:
@uranusjr snap!